The test fails:
`Traceback (most recent call last):
  File "/workspace/pip_test/tests/image_ops_test.py", line 2431, in testTotalVariationHandmade
    self._test(a, tot_var)
  File "/workspace/pip_test/tests/image_ops_test.py", line 2301, in _test
    y = image_ops.total_variation(images=x_tf)
AttributeError: 'module' object has no attribute 'total_variation`

--- a/tensorflow/python/ops/image_ops.py
+++ b/tensorflow/python/ops/image_ops.py
@@ -162,6 +162,10 @@ type and representation (RGB or HSV).
 @@draw_bounding_boxes
 @@non_max_suppression
 @@sample_distorted_bounding_box
+
+## Denoising
+
+@@total_variation
 """
 from __future__ import absolute_import
 from __future__ import division

--- a/tensorflow/python/ops/image_ops_impl.py
+++ b/tensorflow/python/ops/image_ops_impl.py
@@ -1262,7 +1262,7 @@ def decode_image(contents, channels=None, name=None):
 
 
 def total_variation(images, name=None):
-  """Calculate and return the Total Variation for one or more images.
+  """Calculate and return the total variation for one or more images.
 
   The total variation is the sum of the absolute differences for neighboring
   pixel-values in the input images. This measures how much noise is in the

